conversion/conversion.go

```diff
@@ -417,6 +417,70 @@ func getSeekable(f *os.File) (*os.File, int64, error) {
 	return fcopy, n, nil
 }
 
+func VerifyTargetDb(targetDb, project, instance, dbName string) (string, bool, error) {
+	ctx := context.Background()
+	adminClient, err := database.NewDatabaseAdminClient(ctx)
+	if err != nil {
+		return "", false, fmt.Errorf("can't create admin client: %w", analyzeError(err, project, instance))
+	}
+	defer adminClient.Close()
+
+	dbPath := fmt.Sprintf("projects/%s/instances/%s/databases/%s", project, instance, dbName)
+	_, err = adminClient.GetDatabase(ctx, &adminpb.GetDatabaseRequest{Name: dbPath})
+	if err != nil {
+		// Differentiate between "db does not exist" and other errors? Assuming err means db does not exist for now
```

Comment:
Should there be checking on what type of error it is?

Reply:
yes, it's a good practice in general for checking on specific error code and return a more specific error message to the end user